### ISSUE TITLE
Today Extension warning

### ISSUE THREAD

**Issue by nahung89** [schickling/Device.swift#52]
I'm using your library in Today Extension. It works great, but there is a bit problem that XCode always shows this warning: 
```
linking against a dylib which is not safe for use in application extensions: /Users/xxx/Projects/Test/Carthage/Build/iOS/Device.framework/Device
```

I create a test project, which might help you check this issue easier. Also I find out a solution here that might help to fix: [https://stackoverflow.com/a/52995169/2082851](https://stackoverflow.com/a/52995169/2082851)

[Test.zip](https://github.com/schickling/Device.swift/files/2702893/Test.zip)

Hope we could fix it soon. Thank you for a useful library!

**Comment by igorkislyuk** [comment]
Will fix it soon, thanks for opening issue :)

**Comment by igorkislyuk** [comment]
Thank you for solution! Fixed in release 1.2.1
